src/js/components/Button/doc.js

```diff
@@ -251,6 +271,37 @@ export const themeDoc = {
     type: '{}',
     defaultValue: undefined,
   },
+  'button.badge.background': {
+    description: `The background of the badge.`,
+    type: `string | { dark: string, light: string } | {
+      color: { dark: string, light: string } | string,
+      dark: bool,
+      image: string,
+      position: string,
+      opacity: bool | string,
+      repeat: no-repeat | repeat,
+      size: cover | contain | string
+    }`,
+    defaultValue: 'brand',
+  },
+  'button.badge.border': {
+    description:
+      'The border for the badge. Accepts any valid Box border value.',
+    type: 'string | object',
+    defaultValue: undefined,
+  },
+  'button.badge.max': {
+    description: `The max number to display in the badge. If max is 9 and badge 
+    has value of 10, 9+ will be displayed.`,
+    type: 'number',
+    defaultValue: undefined,
+  },
+  'button.badge.size.medium': {
```

Comment:
I'm wondering if we should prepare for eventually having `size` affect the Text size? Since we're going to flex the size a bit based on the content anyway, I'm wondering if we should structure the theme to have `button.badge.text.size` and `button.badge.container.pad`, and move `background` and `border` into `button.badge.container`.

Reply:
I've included this in my latest changes. let me know if it aligns with what your were envisioning.

Reply:
Sounds good for text size, should this `button.badge.size.medium` go in `button.badge.container.width: { min : '24px' }`?

Reply:
It affects height as well. I was mirroring the approach of Avatar